scripts/build_docs.py

```diff
@@ -3,21 +3,25 @@
 
 To use::
 
-  $ python documentation/build_docs.py
+  $ python scripts/build_docs.py
 """
 
 from importlib.machinery import SourceFileLoader
 import os
 from subprocess import Popen
 
-docs_dir = os.path.dirname(os.path.abspath(__file__))
-version_file = os.path.join(docs_dir, os.pardir, "agentos", "version.py")
+scripts_dir = os.path.dirname(os.path.abspath(__file__))
+version_file = os.path.join(scripts_dir, os.pardir, "agentos", "version.py")
 loaded = SourceFileLoader("agentos.version", version_file).load_module()
```

Comment:
Maybe it would be cleaner if we used `agentos.version` here?

Reply:
I'll play around with that.   Should I just be able to `import agentos`  directly given the state of the requirements?

Reply:
ah looks like `agentos.__version__` works.  Is that what you were thinking?

Reply:
Ha, yes, that's what I was thinking.